lighthouse-core/test/audits/seo/font-size-test.js

```diff
@@ -280,6 +281,7 @@ describe('SEO: Font size audit', () => {
         attributes: ['size', '10px'],
       });
 
+      expect(auditResult.details.items[0].source).toBe('http://www.example.com');
```

Comment:
it's also funny because the whole reason I recommended `toMatchObject` in the first place was because you were asserting against `.source` and `.selector` separately 😆

Reply:
done